Fix Android CI builds

```
meson.build:1:0: ERROR: Unknown compiler(s): [['/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android28-clang']]
The following exception(s) were encountered:
Running `/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android28-clang --version` gave "[Errno 2] No such file or directory: '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android28-clang'"

A full log can be found at /home/runner/work/rizin/rizin/build/meson-logs/meson-log.txt
Error: Process completed with exit code 1.
```

See `.github/meson-android-*` files here: https://github.com/rizinorg/rizin/tree/dev/.github

--- a/.github/meson-android-aarch64.ini
+++ b/.github/meson-android-aarch64.ini
@@ -1,15 +1,15 @@
 [binaries]
-c       = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android28-clang'
-cpp     = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android28-clang++'
-ar      = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-ar'
-as      = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android-as'
-ranlib  = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android-ranlib'
-ld      = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android-ld'
-strip   = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android-strip'
+c       = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android33-clang'
+cpp     = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android33-clang++'
+ar      = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-ar'
+as      = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android-as'
+ranlib  = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android-ranlib'
+ld      = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android-ld'
+strip   = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android-strip'
 pkgconfig = 'false'
 
 [properties]
-sys_root = '/usr/local/lib/android/sdk/ndk-bundle/sysroot'
+sys_root = '${ANDROID_NDK}/sysroot'
 
 [host_machine]
 system = 'android'

--- a/.github/meson-android-arm.ini
+++ b/.github/meson-android-arm.ini
@@ -1,15 +1,15 @@
 [binaries]
-c       = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/armv7a-linux-androideabi28-clang'
-cpp     = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/armv7a-linux-androideabi28-clang++'
-ar      = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-ar'
-as      = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/arm-linux-androideabi-as'
-ranlib  = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/arm-linux-androideabi-ranlib'
-ld      = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/arm-linux-androideabi-ld'
-strip   = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/arm-linux-androideabi-strip'
+c       = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/armv7a-linux-androideabi33-clang'
+cpp     = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/armv7a-linux-androideabi33-clang++'
+ar      = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-ar'
+as      = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/arm-linux-androideabi-as'
+ranlib  = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/arm-linux-androideabi-ranlib'
+ld      = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/arm-linux-androideabi-ld'
+strip   = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/arm-linux-androideabi-strip'
 pkgconfig = 'false'
 
 [properties]
-sys_root = '/usr/local/lib/android/sdk/ndk-bundle/sysroot'
+sys_root = '${ANDROID_NDK}/sysroot'
 
 [host_machine]
 system = 'android'

--- a/.github/meson-android-x86_64.ini
+++ b/.github/meson-android-x86_64.ini
@@ -1,15 +1,15 @@
 [binaries]
-c       = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android28-clang'
-cpp     = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android28-clang++'
-ar      = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-ar'
-as      = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android-as'
-ranlib  = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android-ranlib'
-ld      = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android-ld'
-strip   = '/usr/local/lib/android/sdk/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android-strip'
+c       = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android33-clang'
+cpp     = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android33-clang++'
+ar      = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-ar'
+as      = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android-as'
+ranlib  = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android-ranlib'
+ld      = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android-ld'
+strip   = '${ANDROID_NDK}/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android-strip'
 pkgconfig = 'false'
 
 [properties]
-sys_root = '/usr/local/lib/android/sdk/ndk-bundle/sysroot'
+sys_root = '${ANDROID_NDK}/sysroot'
 
 [host_machine]
 system = 'android'

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -653,6 +653,7 @@ jobs:
       - name: Compile with meson
         run: |
           export PATH=${HOME}/.local/bin:${PATH}
+          sed -i 's@\${ANDROID_NDK}@'"${ANDROID_NDK}"'@' .github/meson-android-${{ matrix.name }}.ini
           meson --buildtype release --default-library static --prefix=/tmp/android-dir -Dportable=true -Dblob=true -Dstatic_runtime=true build --cross-file .github/meson-android-${{ matrix.name }}.ini
           ninja -C build && ninja -C build install
       - name: Create rizin-android-${{ matrix.name }}.tar.gz

